common/src/hand_off.rs

```diff
@@ -279,8 +279,10 @@ pub struct FirmwareHandoffTable {
     /// Fuse log Address
     pub fuse_log_addr: u32,
 
+    pub rt_dice_sign: [u8; core::mem::size_of::<caliptra_drivers::Ecc384Signature>()],
```

Comment:
Could the type here just be the Ecc384Signature structure directly? (Although it might require adding the zerocopy trait to a couple types in the driver crate).

Reply:
Correct. This  requires Ecc384Signature, Array4x12 to implement zerocopy as well.

Reply:
Is that something we can do, or is there some concern with doing this?

Reply:
I think it is doable. Let me take a stab at it.

Reply:
AsBytes is not supported with generics.
[11:46](https://caliptraworkspace.slack.com/archives/D04RK3XSKV4/p1686768401422639)
error: unsupported on types with type parameters
  --> drivers/src/array.rs:21:17
   |
21 | #[derive(Debug, AsBytes, Clone, Copy, PartialEq, Eq)]
   |                 ^^^^^^^
   |
   = note: this error originates in the derive macro `AsBytes` (in Nightly builds, run with -Z macro-backtrace for more info)
[11:47](https://caliptraworkspace.slack.com/archives/D04RK3XSKV4/p1686768427178189)
#[repr(transparent)]
#[derive(Debug, AsBytes, Clone, Copy, PartialEq, Eq)]
pub struct Array4xN<const W: usize, const B: usize>(pub [u32; W]);
impl<const W: usize, const B: usize> Array4xN<W, B> {
    pub const fn new(val: [u32; W]) -> Self {
        Self(val)
    }
}
[11:47](https://caliptraworkspace.slack.com/archives/D04RK3XSKV4/p1686768469919159)
/// ECC-384 Coordinate
pub type Ecc384Scalar = Array4x12;